### PR TITLE
Create script to easily bump release

### PR DESCRIPTION
**Pull request by ashangit**

### CODE REVIEW

File: scripts/bump_release.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -e
+set -x
+
+NEW_GARMADON_RELEASE=$1
```
Comment (gray-matter) [inline scripts/bump_release.sh]: Fail if $# < 0 maybe?

File: scripts/bump_release.sh [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+cd "`dirname "$0"`/.."
+
+# Create/Switch to release branch locally
+git checkout release_$NEW_GARMADON_RELEASE || git checkout -b release_$NEW_GARMADON_RELEASE
```
Comment (gray-matter) [inline scripts/bump_release.sh]: What happens if both fail? Shouldn't we exit the script?

File: scripts/bump_release.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+# Bump release
+mvn versions:set -DnewVersion=${NEW_GARMADON_RELEASE}
+find . -name pom.xml.versionsBackup -delete
+sed -i "s#<garmadon.version>.*</garmadon.version>#<garmadon.version>${NEW_GARMADON_RELEASE}</garmadon.version>#g" pom.xml
+
+# Commit release
+git add *pom.xml
```
Comment (gray-matter) [inline scripts/bump_release.sh]: Unnecessary star?

File: scripts/bump_release.sh [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+git commit -a -m "Prepare release ${NEW_GARMADON_RELEASE}"
```
Comment (gray-matter) [inline scripts/bump_release.sh]: What's the point of git add if you then git commit -a?

File: scripts/bump_release.sh (hunk 2 above)
Comment (ashangit) [reply scripts/bump_release.sh]: set -e -> Exit immediately if a command exits with a non-zero status.

Comment (gray-matter) [reply scripts/bump_release.sh]: True that!

**Pull request merged**